### PR TITLE
feat(app): add plugin

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: plugins/app/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+![plugin-app](banner.jpg)
```
Comment (beanow-at-crabnebula) [inline plugins/app/README.md]: Think the banner isn't included.

File: plugins/app/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+<!-- description -->
```
Comment (beanow-at-crabnebula) [inline plugins/app/README.md]: Missing an important: what does it do, what is it for?

**Review by wusyong — COMMENTED** [review]
Do we need `plugins/app/Cargo.lock`? Other plugin doesn't seem to have it.
Beside plugin serves as library. I feel like it should exclude the lock file.

File: plugins/app/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Install
+
+_This plugin requires a Rust version of at least **1.64**_
```
Comment (beanow-at-crabnebula) [inline plugins/app/README.md:7]: Should probably add listing to top-level readme as well.

File: plugins/app/README.md (hunk 1 above)
Comment (lucasfernog) [reply plugins/app/README.md]: We'll need to request it form Alve later for all new plugins

File: plugins/app/README.md (hunk 3 above)
Comment (FabianLars) [reply plugins/app/README.md:7]: If you mean the MSRV, it's already there all the way at the bottom. If you mean the plugin should be added to the table then you're right, completely slipped my mind for the other plugins too...

**Review by FabianLars — APPROVED** [review]

**Pull request merged**